javascript/node/selenium-webdriver/firefox.js

```diff
@@ -166,9 +166,26 @@ async function installExtension(extension, dir) {
   }
 
   let buf = await archive.getFile('manifest.json');
-  let {applications} =
+
+  let {browser_specific_settings} =
+      /** @type {{browser_specific_settings:{gecko:{id:string}}}} */(
+        JSON.parse(buf.toString('utf8')));
```

Comment:
Save the `JSON.parse` result to a variable so you don't have to re-parse on line 185 below.

That, or perhaps change this to:

```js
let {applications, browser_specific_settings} = JSON.parse(buf.toString('utf8'))
```